src/main/java/de/tum/in/www1/artemis/repository/ProgrammingExerciseRepository.java

```diff
@@ -167,31 +174,31 @@ AND EXISTS (SELECT r.assessor FROM s.results r
     @Query("""
             SELECT COUNT (DISTINCT p) FROM ProgrammingExerciseStudentParticipation p
             WHERE p.exercise.id = :#{#exerciseId}
+            AND (p.testRun = FALSE OR p.testRun IS NULL)
```

Comment:
```suggestion
            AND p.testRun = FALSE
```

Reply:
fixed in 139c8fb547deadc2b10c0e497d91389dcb5f736c